cmd/manager/server.go

```diff
@@ -123,12 +118,6 @@ func run() {
 		log.Fatalf("Could not add all controllers to operator manager: %v", err)
 	}
 
-	// Create Service object to expose the metrics port.
-	_, err = metrics.CreateMetricsService(ctx, cfg, metricsServicePort)
-	if err != nil {
-		log.Errorf("Could not create a service to expose the metrics port: %v", err.Error())
-	}
-
```

Comment:
Don't we still need to expose the metrics port?

Reply:
This is just creating a Service I think. We can just do the same in `deploy/`?

Reply:
Yes, I think so.

Reply:
Adding it to deploy